src/graphics-private.h

```diff
@@ -105,8 +106,12 @@ typedef struct _Graphics {
 	cairo_t			*ct;
 	GpMatrix		*copy_of_ctm;
 	cairo_matrix_t		previous_matrix;
+#if HAS_X11
+#ifdef CAIRO_HAS_XLIB_SURFACE
 	Display			*display;
 	Drawable		drawable;
+#endif
+#endif
```

Comment:
I checked, it's not in [gdipStructs.cs](https://github.com/mono/mono/blob/master/mcs/class/System.Drawing/System.Drawing/gdipStructs.cs) and a search for `_Graphics` didn't return any struct which matches this one in Mono's codebase, so I'm assuming not.